cmd/template/cluster/flag.go

```diff
@@ -73,6 +75,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&f.ExternalSNAT, flagExternalSNAT, false, "AWS CNI configuration.")
 	cmd.Flags().StringVar(&f.PodsCIDR, flagPodsCIDR, "", "CIDR used for the pods.")
 	cmd.Flags().StringVar(&f.Credential, flagCredential, "credential-default", "Cloud provider credentials used to spin up the cluster.")
+	cmd.Flags().StringVar(&f.NetworkPool, flagNetworkPool, "", "NetworkPool for the subnets of the nodes.")
```

Comment:
Should we add some example so it is easier for at least me to picture that? What would be a legitimate example? 😄 

```suggestion
	cmd.Flags().StringVar(&f.NetworkPool, flagNetworkPool, "", "NetworkPool for the subnets of the nodes, e.g. 127.0.0.1/16.")
```

Reply:
This flag references a "name" so maybe I can just say that in the help "Name of the NetworkPool that will be used for the subnets of the nodes"